setup.cfg

```diff
@@ -25,14 +26,15 @@ install_requires =
     nibabel ~= 3.0
     nipype ~= 1.4
     nitime
+    nitransforms >= 20.0.0rc3,<20.2
```

Comment:
should we pin this to master as well?

Reply:
we'll need to set a release pin to cut the RC, so I'd leave this. 20.2 should be tolerant enough to allow nitransforms progress without breaking anything used in here.